src/models/MediaImage.ts

```diff
@@ -198,6 +198,12 @@ export class MediaImageL {
     onRightClick = (): void => {}
 
     onClick = (ev: MouseEvent): void => {
+        //Delete if not starred
+        if (ev.ctrlKey && ev.shiftKey && ev.altKey && !this.star) {
+            ev.stopPropagation()
+            ev.preventDefault()
+            return void this.delete()
+        }
```

Comment:
This should be done through the command/action system, not the underlying widget's event as this propagates to every MediaImageL instead of just the ones in the Gallery.
rvion would need to say how to do the proper way to do polling or context gathering with the system though, I'm unsure how to do it and the only example for images (cmd_copyImage) is still for *every* image instead of just the ones in the Gallery.

The shortcut(s) for this should just be X and/or Delete and then check for the hovered image in the Gallery. Way less things to press/hold down, although I think there should be some sort of selection system probably.

Reply:
Agreed, this was more of a hack because I wanted something and I could copy the other clicks that were already there. I initially tried what you said with the delete button. But when I added a keyboard command it was getting intercepted somewhere and I am just not quite smart enough on events and javascript in general to work it out. @rvion please magix?